examples/petsc/bps.c

```diff
@@ -489,6 +444,10 @@ int main(int argc, char **argv) {
   ierr = PetscOptionsBool("-write_solution", "Write solution for visualization",
                           NULL, rp->write_solution, &rp->write_solution, NULL);
   CHKERRQ(ierr);
+  rp->simplex = PETSC_FALSE;
+  ierr = PetscOptionsBool("-simplex", "Element topology (default:hex)",
+                          NULL, rp->simplex, &rp->simplex, NULL);
+  CHKERRQ(ierr);
```

Comment:
I'd check here if you have BP5/6 + simplex elements. This is the first place you can fail with that knowledge

Reply:
I will add for other problems as well (multigrid, area, ...)

Reply:
What problem should I choose for test? Right now for bps we have
`//TESTARGS -ceed {ceed_resource} -test -problem bp5 -degree 3 -ksp_max_it_clip 50,50 -simplex`
any preference?

Reply:
BP3 would work for CI